ui/ducks/metamask/metamask.test.js

```diff
@@ -37,6 +47,64 @@ describe('MetaMask Reducers', () => {
             name: 'Send Account 4',
           },
         },
+        internalAccounts: {
+          accounts: {
+            'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3': {
+              address: '0xfdea65c8e26263f6d9a1b5de9555d2931a33b825',
+              id: 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3',
+              metadata: {
+                name: 'Send Account 1',
+                keyring: {
+                  type: 'HD Key Tree',
+                },
+              },
+              options: {},
+              methods: EOA_EVM_METHODS,
```

Comment:
Just a question, is there a reason to have this array in this test and in the others have the methods as a destructure of `EthMethod`?

Reply:
It should be migrated to EOA_EVM_METHODS, the keyring api will return user op methods as well.